src/main.rs

```diff
@@ -173,7 +173,10 @@ fn cli_init() -> Result<ValidatedArgs> {
 
 fn main_impl() -> Result<()> {
     let args = cli_init()?;
+    #[cfg(not(feature = "passthru_ahk"))]
```

Comment:
I don't think this flag should be necessary here.

This feature should create conditionally compile another `new_x` function that leaves this code untouched, like how `new_arc` wraps `new`.